openhabian-setup.sh

```diff
@@ -1614,6 +1637,7 @@ show_main_menu() {
     "25 | Homegear"     "Homematic specific, the CCU2 emulation software Homegear" \
     "26 | knxd"         "KNX specific, the KNX router/gateway daemon knxd" \
     "27 | 1wire"        "1wire specific, owserver and related packages" \
+    "28 | NodeRED"      "Flow-based programming for the Internet of Things" \
```

Comment:
Come to think about it, let's move it up a bit, in front of the hardware-related components. I don't know why 24 is missing in this list but maybe that's a sign ;)